src/complete_hack.py

```diff
@@ -36,6 +36,9 @@ def _check_pending_node(self):
         if now > expires:
             self._pending_node['status'] = "completed"
             self._db.submit({'node': self._pending_node})
+            checkout_node = self._db.get_node(self._pending_node['parent'])
```

Comment:
I don't think this is needed here anymore.
https://github.com/kernelci/kernelci-pipeline/pull/66 will take care of setting the node status to 'complete'.